src/watertap_contrib/seto/unit_models/surrogate/tests/test_lt_med_surrogate.py

```diff
@@ -43,35 +43,131 @@ def LT_MED_frame(self):
         m.fs.water_prop = SeawaterParameterBlock()
```

Comment:
Should have a short build test for the ``ConfigurationError`` in both these test files.

Reply:
A test for the `ValueError` raised by IDAES was added.